src/httpc.h

```diff
@@ -439,6 +439,16 @@ httpc_set_follow_location(struct httpc_request *req, long follow);
 void
 httpc_set_accept_encoding(struct httpc_request *req, const char *encoding);
 
+/**
+ * Specify http protocol version for request
+ * @param req request
+ * @param version - http version to use for this request
+ * Version can be 1.0, 1.1, 2, 2-tls or 2-prior-knowledge
```

Comment:
we decided to remove http version 1.0, please update comment

Reply:
Fixed